client/src/styles/OrderField.scss

```diff
@@ -0,0 +1,13 @@
+.ckan-resource__order.field .form__field-holder {
+  input {
+      max-width: 6em;
+      min-width: 3em;
+      display: inline-block;
+  }
+
+  .form__field-description {
+    display: inline-block;
+    font-size: $font-size-root;
+    margin-left: 0.5em;
```

Comment:
There's a variable you can use for this somewhere I think, it's usually 0.5385rem or something

Reply:
It's scary how accurate you are sometimes
![image](https://user-images.githubusercontent.com/778003/50310670-8ffad080-0507-11e9-8e91-3ce54c4a98ed.png)